cmd/proxy/Dockerfile

```diff
@@ -27,19 +27,17 @@ RUN DATE="$(date -u +%Y-%m-%d-%H:%M:%S-%Z)" && \
     -ldflags "-X github.com/gomods/athens/pkg/build.version=$VERSION -X github.com/gomods/athens/pkg/build.buildDate=$DATE -s -w" \
     -o /bin/athens-proxy ./cmd/proxy
 
-FROM alpine:${ALPINE_VERSION}
+FROM golang:${GOLANG_VERSION}-alpine
```

Comment:
So, I think if we just set the values for GOROOT=/go, GOSUMDB=off, and GOPROXY=direct then it'll fix this issue.

Reply:
The only thing I'm not super sure of is if our users would find turning the sumdb off and proxy to direct a sane default. Maybe we can figure out what it used to default to?

Reply:
This does not work, you must move the GOROOT through the code, otherwise you get the same error.

I will make the needed changes, so we have a full comparison, but my concerns are as follows:
1. It is not obvious that GOROOT needs to move through the code, and will likely be missed down the line
2. I doubt this is the last rodio we will have with moving the Go bin around like this, and breaks are a little untestable.